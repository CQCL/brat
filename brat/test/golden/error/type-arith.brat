f(n :: #, Vec(Nat, n ^ 3)) -> Bool
f = ?f
