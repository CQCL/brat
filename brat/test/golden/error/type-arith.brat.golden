Error in test/golden/error/type-arith.brat@1:20
f(n :: #, Vec(Nat, n ^ 3)) -> Bool
                   ^^^^^

  Type error: Type level arithmetic too confusing

